sysutils/ctop/Portfile

```diff
@@ -18,9 +18,10 @@ long_description    \
     future releases.
 homepage            https://ctop.sh
 
-checksums           rmd160  8743fb5fd970399665c5ecc9ec885635af495e11 \
-                    sha256  b191ffa8ba343edb47644fcbc57868a282a4bd720b1836f8e0ec15820880d23d \
-                    size    1291305
+checksums           ${distname}${extract.suffix} \
```

Comment:
I don't think the distname line is necessary; it's the default and there is only one file, right?

Reply:
Thanks, `go2port` inserts it by default during an update and I forgot to delete it.